translations/monero-core_it.ts

```diff
@@ -784,27 +785,27 @@
     <message>
         <location filename="../pages/Settings.qml" line="525"/>
         <source>Wallet seed &amp; keys</source>
-        <translation type="unfinished"></translation>
+        <translation>Wallet seed &amp; chiavi</translation>
```

Comment:
I think we can translate `Wallet` to `Portafoglio` here

Reply:
Seed &amp; chiavi portafoglio is it ok?